frontend/src/api/reducers/sessions.js

```diff
@@ -9,7 +9,7 @@ import { createBasicReducerObject, createReducer } from "./utils";
 
 const initialState = {
     _modelData: [],
-    activeSession: { id: null }
+    activeSession: { id: null, name: "" }
```

Comment:
Is there not an active session selector? I don't think the name should be stored in multiple places...

Reply:
I agree that that it should not be stored in multiple places. The reason why I added this here is because there was a warning complaining about missing `activeSession.name` when active session is unset. That is due to the fact that `name` is a required attribute for session.  IMHO, it make sense to have the initial value contain all the required fields (maybe we can have a util function where it automatically create all the required fields, it can even have some flexibility where you can specify the value for some particular fields). 

I think instead of removing `name` here, it makes more sense to remove the ` = { id: null, name: "" }` of 
```
 const {
        sessions = [],
        activeSession = { id: null, name: "" },
        setActiveSession
    } = props;
```
in active session selector component as `activeSession` would not be empty; it would at least be the default value, and it has the name field as well.

Reply:
The proptype should null or session